### DISCUSSION TITLE
Using Qusasar dev server with Navigo.js for routing

### DISCUSSION THREAD

**Discussion by JonJC** [quasarframework/quasar#10631] [General - Components / Directives / etc]
I'm trying to use Quasar but with Navigo.js to handle the routing (I'm not entirely sure this is a good idea, but I'm trying it on the recommendation of a course as a way to be able to abstract out routing and navigation from the framework to aid with testability and portability). I have no problem being able to use Navigo to change the url in the browser and then detect this change and update the shown Vue component. Where I'm struggling is when the user enters a valid url in the browser - I get a 404 - Cannot GET /app/myPath
Looking here https://github.com/krasimir/navigo/issues/230 I see that my problem is probably that I need to 'configure your web server to point to your index.html page regardless of the URL path.'
My question is: how can I do that? - I'm curently using the dev server (so starting with 'quasar dev').
I've looked through: https://v0-17.quasar-framework.org/guide/app-quasar.conf.js.html and https://webpack.js.org/configuration/dev-server/#devserveronaftersetupmiddleware but don't see anything relevant.
One possibility I've considered is setting up standard Vue routing, but not actually using it - on the assumption that doing so would register those URLs with the dev server. I thought I'd ask before playing around anymore!
Many thanks,
Jon

**Comment by ontwikkelfabriek** [comment]
@JonJC 

Asking if it is possible to decouple Quasar from the **official** Vue router to use another router, is the most niche question ever posted. Congrats! 🥇  

 On a serious note, I would ask this at Quasar's discord serve if you **really** want to know  if this is possible. (and how to do that)
